plugins/eccube-dev-agents/skills/reply-review/SKILL.md

```
@@ -1,88 +1,101 @@
   -X POST \
   -f body="返信内容" \
   -F in_reply_to={comment_id}
 ```
 
-複数コメントがある場合はそれぞれに返信を投稿し、処理結果を一覧で報告。
+複数コメントがある場合はそれぞれのスレッドに返信を投稿する。
 
-### 7. 結果報告
+### 6. 結果報告
 
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)